### DISCUSSION TITLE
Docker Compose is being moved to Docker CLI

### DISCUSSION THREAD

**Discussion by danielsuguimoto** [kool-dev/kool#337] [Ideas]
Docker-compose commands are now under Docker CLI on new versions...`kool start`, for example, can call `docker compose up`....but it must be compatible with users with older Docker versions...

**Comment by fabriciojs** [comment]
Awesome, yeah we are keeping track of that.

Right now, `docker-compose` is already an optional installation, since in case it is not available in the host OS, we use a container to run `docker-compose`. This verification right now is done merely based on the LookPath for `docker-compose` executable.

We can think of a straightforward implementation where we should assert `docker` version and then decide if we should use `docker compose` or fall back to the current existing behavior.
